### ISSUE TITLE
Parameters not working for liking/following etc.

### ISSUE THREAD

**Issue by youngkm3** [ohld/igbot#552]
## Please follow the guide below

- Issues submitted without this template format will be **ignored**.
- You will be asked some questions and requested to provide some information, please read them **carefully** and answer completely.
- Put an `x` into all the boxes [ ] relevant to your issue (like so [x]).
- Use the *Preview* tab to see how your issue will actually look like.

---

### Before submitting an issue, make sure you have:
- [x ] Updated to the latest version
- [x ] Read the [README](https://github.com/instagrambot/instabot/blob/master/README.md) and [docs](https://github.com/instagrambot/instabot/tree/master/docs)
- [ x] [Searched](https://github.com/instagrambot/instabot/search?type=Issues) the bugtracker for similar issues including **closed** ones

### Purpose of your issue?
- [ x] Bug report (encountered problems/errors)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### The following sections requests more details for particular types of issues, you can remove any section (the contents between the triple ---) not applicable to your issue.

---

### For a *bug report*, you **must** include the Python version used, *code* that will reproduce the error, and the *error log/traceback*.

Paste the output of ``python -V`` here: 3.6.5

### Describe your issue

**I am running the "multi_script_CLI.py" example and it doesn't seem that my parameters are being considered when I use the "follow_users_by_hashtag" method.  It keeps giving me errors on about 80% of accounts saying that it is skipping.  Also, I want to follow Business accounts.  See the log below:**

```
follower_count > bot.max_followers_to_follow, skipping!
Adding '1822361236' to `skipped.txt`.
Processed users:  48%|████████████████████████▏                         | 31/64 [05:36<05:57, 10.85s/it]  ===> Going to follow `user_id`: 6623787497.

USER_NAME: rochaxtreme, FOLLOWER: 191, FOLLOWING: 783

info: is BUSINESS, skipping!
Adding '6623787497' to `skipped.txt`.
Processed users:  50%|█████████████████████████                         | 32/64 [05:39<05:39, 10.61s/it]  ===> Going to follow `user_id`: 3264570547.

USER_NAME: popaisnack, FOLLOWER: 9666, FOLLOWING: 1906

following_count / follower_count > bot.max_following_to_followers_ratio, skipping!
Adding '1695901315' to `skipped.txt`.
Processed users:  46%|███████████████████████                           | 24/52 [02:45<03:13,  6.91s/it]  ===> Going to follow `user_id`: 1741231574.

USER_NAME: nurdascimaja, FOLLOWER: 626, FOLLOWING: 683

following_count > bot.max_following_to_follow, skipping!
Adding '47184803' to `skipped.txt`.
Processed users:  96%|████████████████████████████████████████████████▏ | 54/56 [08:44<00:19,  9.71s/it]  ===> Going to follow `user_id`: 200743258.

USER_NAME: cherylmaas, FOLLOWER: 19951, FOLLOWING: 326

info: is VERIFIED, skipping !
Adding '200743258' to `skipped.txt`.
Processed users:  98%|█████████████████████████████████████████████████ | 55/56 [08:51<00:09,  9.67s/it]  ===> Going to follow `user_id`: 4091545253.

USER_NAME: mr_8pack, FOLLOWER: 226, FOLLOWING: 27
```

My parameters are set up as follows (there should be no way that any of the filters are hit based off my high numbers):
```
Current parameters

Max likes per day: 1000

Max unlikes per day: 1

Max follows per day: 500

Max unfollows per day: 1

Max comments per day: 1

Max likes to like: 9999999

Max followers to follow: 9999999

Min followers to follow: 0

Max following to follow: 10000

Min following to follow: 0

Max followers to following_ratio: 9999999

Max following to followers_ratio: 9999999

Min media_count to follow:0

Like delay: 1

Unlike delay: 1

Follow delay: 5

Unfollow delay: 5

Comment delay: 5

Proxy: None

```

So in summary, I'm basically looking to follow ALL accounts with certain hashtags, but 80% of the accounts get skipped either due to being Business account, not meeting parameters, or being verified.  

Please help!  Thanks

**Comment by valentinpompa** [comment]
Use "filter_users=False" in Bot section.

Example: 

`bot = Bot(filter_users=False, like_delay=180, max_likes_to_like=80)`

Then you can have that error.
